docs/application/dotnet/get-started/certificates/index.md

```diff
@@ -9,12 +9,12 @@ The certification process is necessary not only for testing your apps on the act
 The certification process guarantees that your application will not be distributed by someone else even if your application package is leaked.
 The following pages describe how to generate certificates, how to maintain them, and how to package your application:
 
-- [Installing Certificates Extension](installing-certificate-extension.md)
+- [Installing the Extension](installing-the-extension.md)
```

Comment:
I have a question. Do we really need to change the URLs of existing site pages?
URLs can be used on other sites (e.g. developer.samsung.com). If the change is inevitable, please consider website redirection (pointing the existing URL to a new page, so the existing URL and the new one can coexist.)

Reply:
The changes in the content (some pages were moved) are based on the guide (designers conducted UX audit and users interviews). We maintain a list with old and new URLs of moved articles. We will send (once this change will be accepted) it to the server owners so proper redirections in the web server could be done .